kubernetes/README.adoc

```diff
@@ -0,0 +1,56 @@
+= Kubernetes and Openshift tutorial
+
+This tutorial contains an instruction how to run Infinispan library mode (as a microservice) in Kubernetes/OpenShift.
+
+Prerequisites: Maven and Docker Deaemon running in the background.
+
+== Setting up OpenShift
+
+OpenShift Origin has a command line client called `oc` (it can be downloaded https://github.com/openshift/origin/releases[here]).
+
```

Comment:
IMO it's better to put a link directly to https://github.com/openshift/origin/releases